pkg/operator/endpoints/logs.go

```diff
@@ -56,15 +55,15 @@ func ReadLogs(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		if workloadID == "" {
-			RespondError(w, errors.New(appName, "latest workload ID", resourceID, s.ErrNotFound))
+			RespondError(w, errors.Wrap(workloads.ErrorNotFound(), appName, "latest workload ID", resourceID))
 			return
 		}
 		readLogs(w, r, workloadID, appName, verbose)
 		return
 	}
 
 	if resourceName == "" {
-		RespondError(w, errors.New(s.ErrAnyQueryParamMustBeProvided("workloadID", "resourceID", "resourceName")))
+		RespondError(w, ErrorPathParamRequired("workloadID", "resourceID", "resourceName"))
```

Comment:
This should be query param. Also it should probably be a separate error, so it's clear that it's joined by "or" (e.g. it could be called `ErrAnyQueryParamRequired`)